examples/_buildParamTypes.py

```diff
@@ -69,8 +64,8 @@ def makeAllParamTypes():
     # Also minor tweak to meta opts
     meta = params.child('Applies to All Types')
     infoChild = meta.child('Extra Information')
-    for ch in meta.children()[1:]:
-        ch.sigValueChanged.connect(lambda _param, _val: infoChild.setOpts(**{_param.name(): _val}))
+    for child in meta.children()[1:]:
+        child.sigValueChanged.connect(lambda _param, _val: infoChild.setOpts(**{_param.name(): _val}))
```

Comment:
@outofculture I know we've had trouble with signals being connected the lambda functions, I'm having a nagging memory that there was a 3rd factor that had to meet, like the lambda function had to reference `self`?  Or am I remembering wrong?

Reply:
Correct. Lambdas are fine so long as they don't reference `self`.